src/ducks/apps/index.js

```diff
@@ -149,25 +149,32 @@ function _sortAlphabetically (array, property) {
   return array.sort((a, b) => a[property] > b[property])
 }
 
+/* Only for the icon fetching */
+const root = document.querySelector('[role=application]')
+const data = root && root.dataset
+const COZY_TOKEN = data && data.cozyToken
+const COZY_DOMAIN = data && (`//${data.cozyDomain}`)
+/* Only for the icon fetching */
+
 async function _getIcon (url) {
   if (!url) return ''
   let icon
   try {
-    icon = await cozy.client.fetchJSON('GET', url)
+    const resp = await fetch(`${COZY_DOMAIN}${url}`, {
+      method: 'GET',
```

Comment:
Don't a `HEAD` call can do the same (I am asking because I am not sure)

Reply:
I don't know, maybe it could work by the `HEAD` is not designed for that according to [the spec](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.4). It's mainly used to get route informations, headers or meta data quickly.

Reply:
As we deal directly with the response, `HEAD` is actually not efficient. 👍